Framework/jmaqs-webservices/src/main/java/com/magenic/jmaqs/webservices/WebServiceTestObject.java

```diff
@@ -13,42 +13,36 @@
 public class WebServiceTestObject extends BaseTestObject {
 
   /**
-   * The HTTP client wrapper.
+   * The Web Service Driver.
    */
-  protected HttpClientWrapper webServiceWrapper;
+  private WebServiceDriver webServiceDriver;
   
   /**
    * Initializes a new instance of the WebServiceTestObject.
-   * 
-   * @param webServiceWrapper
-   *          The web service wrapper
-   * @param logger
-   *          The Logger Object
-   * @param fullyQualifiedTestName
-   *          The fully qualified test name
+   * @param newWebServiceDriver The web service driver.
+   * @param logger The Logger Object.
+   * @param fullyQualifiedTestName The fully qualified test name.
    */
-  public WebServiceTestObject(HttpClientWrapper webServiceWrapper, Logger logger, String fullyQualifiedTestName) {
+  public WebServiceTestObject(final WebServiceDriver newWebServiceDriver,
```

Comment:
I don't think it's necessary to declare param as final. @jason-edstrom thoughts?